gdal/gdal_merge.xml

```diff
@@ -0,0 +1,211 @@
+<tool id="gdal_merge" name="GDAL Merge" version="0.1.0">
+    <description>mosaics a set of images</description>
+    <macros>
+        <import>gdal_macros.xml</import>
+    </macros>   
+    <expand macro="gdal_requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        gdal_merge.py
+            -q        
+            #for $i, $s in enumerate( $series )
+                '${s.input}'
+            #end for
+            
+            -of '$of'
+            
+            #if $settings.advanced=='advanced'
+                
+                #if $settings.co!=''
```

Comment:
I think you need to use `#if str($settings.co) != ''`

Reply:
should i use str() for every text param ?

Reply:
Forget that, this should be ok. But keep in mind that a user could add a space, is this valid?